oidc-exchange.py

```diff
@@ -17,6 +17,12 @@
 
 {message}
 
+You're seeing this because the action wasn't given the inputs needed to
+perform password-based or token-based authentication. If you intended to
+perform one of those authentication methods instead of trusted
+publishing, then you should double-check your secret configuration and variable
+names.
+
```

Comment:
LGTM

Reply:
@joe733 FYI you can send code suggestions instead of the diff blocks and then, the PR author can accept them in one click.

Reply:
Closing this conversation to unblock merging.